WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -213,7 +218,7 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
 
     /// Update the `GhostOptions` to correctly show compact or default cells
     private func updateGhostableTableViewOptions() {
-        let ghostOptions = GhostOptions(displaysSectionHeader: false, reuseIdentifier: postCellIdentifier, rowsPerSection: [10])
+        let ghostOptions = GhostOptions(displaysSectionHeader: false, reuseIdentifier: postCellIdentifier, rowsPerSection: [50])
```

Comment:
Curious: why do we need to add more rows here?

Reply:
+1

Reply:
One image can explain:

<img src="https://user-images.githubusercontent.com/7040243/93632811-56991280-f9c4-11ea-8141-9b9debe32b14.PNG" width="500">

50 is a lot, but given that rows are rendered only when they appear on screen I just added more so we can be better prepared even for an upcoming Apple device with a huge screen. :P

Reply:
😬